filters/openpolicyagent/openpolicyagent.go

```diff
@@ -553,11 +583,7 @@ func (opa *OpenPolicyAgentInstance) startSpanFromContextWithTracer(tr opentracin
 		span = tracing.CreateSpan("open-policy-agent", ctx, tr)
```

Comment:
Can we move this span name to the const as well?

Reply:
Absolutely!